spec/models/queue_item_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe QueueItem do 
+  it { should belong_to(:user) }
+  it { should belong_to(:video) }
+  
+  describe '#video_title' do 
```

Comment:
Still trying to figure this one out :). I please checkout my refactored commit prior to merging. Thanks!